tests/comfy_cli/command/test_workflow_saved.py

```
@@ -531,3 +531,37 @@ def test_404_surfaces_workflow_not_found(self, cloud_target, monkeypatch, capsys
+
+# A non-empty 200 body the client can't decode as JSON: a valid-UTF-8 non-JSON page
+# (e.g. an HTML proxy/error page returned 200) and a non-UTF-8 byte string. The
+# latter makes ``json.loads`` raise ``UnicodeDecodeError`` (not ``JSONDecodeError``),
+# so both must be caught. Neither may be reported as "no data" (empty list / null id).
+_UNPARSEABLE_BODIES = [
+    pytest.param(b"<html>502 Bad Gateway</html>", id="non-json"),
+    pytest.param(b"\xff\xfe\x00bad", id="non-utf8"),
+]
+
+
+class TestUnparseableResponse:
```

Pylint (4.0.6)

[convention] Missing class docstring

(C0115)